utilities/virt.py

```
@@ -2146,26 +2146,63 @@ def vm_instance_from_template(
 
 
 @contextmanager
-def node_mgmt_console(admin_client, node, node_mgmt):
+def _uncordon_and_stabilize(admin_client, node, hco_namespace):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_uncordon_and_stabilize`

Add return type annotation: `None`

(ANN202)